shopgate/pwa
# Fixed product list updates when filters where applied or changed

# Description
This ticket fixes a bug within the `getProductsResult` selector which caused, that product lists where not updated when filters where applied and modified. This issue should be solved now.

## Type of change
- [  ] New Feature :rocket: (non-breaking change which adds functionality)
- [  ] Breaking Change :boom: (fix or feature that would cause existing functionality to not work as expected)
- [  ] Polish :nail_care: (Just some cleanups)
- [  ] Docs :memo: (Changes in the documentations)
- [  ] Internal :house: Only relates to internal processes.

## How to test it
- goto the category "4. all products" within your testshop or search for "Product"
- open the filters and pick both possible values for "Battery Type"
- apply the filters and verify that you can see two products
- remove the filters one after another and check if the product list updates as expected (one product all products)
